### PR TITLE
Add mne-python tutorial

### PR DESCRIPTION
**Pull request by mainakjas**
cc @jackparmer

### CODE REVIEW

**Comment by mainakjas** [comment]
There was also a bug here which I fixed: https://github.com/plotly/IPython-plotly/blob/master/requirements.txt#L3

File: notebooks/mne-tutorial/config.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "title": "Plotly visualizations for MNE-Python \n to process MEG/EEG data",
+    "title_short": "Plotly visualizations for MNE-Python",
+    "meta_description": "Create interactive visualizations using MNE-Python and Plotly",
+    "cells": [0, "end"],
```
Comment (etpinard) [inline notebooks/mne-tutorial/config.json]: `"cells": [1, "end"]` would be better here - to not have the your title appear twice in the published version.

**Comment by etpinard** [comment]
Looking good, but your seem to be missing the published `body.html` and `config.json` files.

**Comment by mainakjas** [comment]
ok I added those two files. But I just noticed that the `html` file is building only two cells. Any clue? Here's a screenshot: 
weird_html

**Comment by jackparmer** [comment]
Nice! +1 from me

**Comment by etpinard** [comment]
@mainakjas Looks like you're another victim of bug https://github.com/plotly/IPython-plotly/issues/25.

I'll take it from here.

Thanks again for your efforts.

**Comment by mainakjas** [comment]
Great, thanks and do let me know when it's live :)

**Comment by etpinard** [comment]
https://plot.ly/ipython-notebooks/mne-tutorial/

**Comment by mainakjas** [comment]
For Out [15] I don't see the controls for zooming in / out

**Comment by etpinard** [comment]
Hmm. Right sorry about that. 

The published version uses a 750px width. 

You can either resize the graph to a 750px width or add in a keyword argument to `py.iplot` as done [here](https://plot.ly/ipython-notebooks/bioinformatics/#In-[53]). 

We are currently working on an improved environment for serving notebooks; formatting issues like these will be addressed shortly.

**Comment by mainakjas** [comment]
I see. Do you want to fix this? Should I make a PR or is it easier for you fix directly on master? I'm not quite sure if I'm able to build the published html

**Comment by etpinard** [comment]
@mainakjas 

You can either:

- add in keyword argument `width` to `py.iplot` in the original notebook, run `make run-publish nb="mne-tutorial"` and make a PR here.

- Or, wait one or two weeks for us to deploy the updated environment. 

your call.

**Comment by mainakjas** [comment]
okay, I will wait. More pressing things to do right now. Thanks ! :)

**Pull request merged**